flower_shop/models/stock_lot.py

```diff
@@ -0,0 +1,34 @@
+from odoo import fields, models, api
+from datetime import datetime, timedelta
+from odoo.exceptions import ValidationError
+
+
+class StockLot(models.Model):
+    _inherit = 'stock.lot'
+
+    flower_water_id = fields.One2many('flower.water', 'lot_id')

```

Comment:
2many fields (One2many or Many2many) should use the following naming convention:
**{MODEL_NAME}_ids** example: **flower_water_ids**